src/test/javascript/unit/factories.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('factories unit testing', function(){
+
```

Comment:
If it is common place for testing all factories then name can remain as is, but if it is only for testing restFactory, maybe it would be better to change description to 'rest factory unit testing'